lib/dialogs/share/index.jsx

```diff
@@ -85,15 +85,11 @@ export class ShareDialog extends Component {
   };
 
   gravatarURL = email => {
-    const hash = createHash('md5');
-
-    hash.update(email.trim().toLowerCase());
-    let digest = hash
-      .digest()
+    let hash = md5(email.trim().toLowerCase())
```

Comment:
`const`? (I didn't realize we don't have the prefer const rule enabled in eslint!)

Reply:
I need to add that! I think I'll merge and then add that rule and the fixes in another pr